gdextension/extension_api.json

```diff
@@ -5,7 +5,7 @@
 		"version_patch": 0,
 		"version_status": "stable",
 		"version_build": "official",
-		"version_full_name": "Godot Engine v4.3.stable.official"
+		"version_full_name": "Blazium Engine v0.1.0.nightly (base v4.3.stable)"
```

Comment:
is this used anywhere?
Also, how will we do versioning here i wonder, we use nightly?

Reply:
This one is generated automatically from build, so when we release our stable version we will use the terminal to generate this file again and we update it like:

`./"blazium Engine v4.3.stable.official.exe" --dump-extension-api --dump-gdextension-interface`